diff --git a/common/autoconf/configure.ac b/common/autoconf/configure.ac
index 7a28f00de46..5e1e3c49ded 100644
--- a/common/autoconf/configure.ac
+++ b/common/autoconf/configure.ac
@@ -191,7 +191,7 @@ LIB_SETUP_MISC_LIBS
 LIB_SETUP_STATIC_LINK_LIBSTDCPP
 
 #PLATFORM_WINMD = /AI"E:\Programme\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.10.25017\lib\x86\store\references"
-PLATFORM_WINMD='/AI"E:\Programme\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.10.25017\lib\x86\store\references"'
+PLATFORM_WINMD='/AI"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references"'
 AC_SUBST(PLATFORM_WINMD)
 AC_ARG_VAR(PLATFORM_WINMD, "The compiler instruction to find the platform.winmd file")
 
diff --git a/common/autoconf/toolchain.m4 b/common/autoconf/toolchain.m4
index 3ea272b6372..dd2f07fe726 100644
--- a/common/autoconf/toolchain.m4
+++ b/common/autoconf/toolchain.m4
@@ -1068,7 +1068,7 @@ AC_DEFUN_ONCE([TOOLCHAIN_SETUP_COMPILER_FLAGS_FOR_JDK],
     cl )
 		# Unfortunately we have to use -MD over -MT because of -Zw
       COMMON_CCXXFLAGS_JDK="$COMMON_CCXXFLAGS $COMMON_CCXXFLAGS_JDK -Zi -MD -Zc:wchar_t- -W3 -wd4800 \
-	  -AI'E:\Programme\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.10.25017\lib\x86\store\references' \
+	  -AI'C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references' \
       -D_STATIC_CPPLIB -D_DISABLE_DEPRECATE_STATIC_CPPLIB -DWIN32_LEAN_AND_MEAN \
       -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE \
       -DWIN32 -DIAL"
diff --git a/hotspot/make/windows/makefiles/adlc.make b/hotspot/make/windows/makefiles/adlc.make
index ea73b2724ac..7fa5334a030 100644
--- a/hotspot/make/windows/makefiles/adlc.make
+++ b/hotspot/make/windows/makefiles/adlc.make
@@ -91,7 +91,7 @@ GENERATED_NAMES_IN_DIR=\
   $(AdlcOutDir)\adGlobals_$(Platform_arch_model).hpp \
   $(AdlcOutDir)\dfa_$(Platform_arch_model).cpp
 
-PLATFORM_WINMD = /AI"E:\Programme\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.10.25017\lib\x86\store\references"
+PLATFORM_WINMD = /AI"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references"
 
 {$(WorkSpace)\src\share\vm\adlc}.cpp.obj::
         $(CXX) $(ADLC_CXX_FLAGS) $(EXH_FLAGS) $(PLATFORM_WINMD) $(CXX_INCLUDE_DIRS) /c $<
diff --git a/hotspot/make/windows/makefiles/compile.make b/hotspot/make/windows/makefiles/compile.make
index 5d3ad15a999..73114e89875 100644
--- a/hotspot/make/windows/makefiles/compile.make
+++ b/hotspot/make/windows/makefiles/compile.make
@@ -164,7 +164,7 @@ MS_RUNTIME_OPTION = /MD
 STATIC_CPPLIB_OPTION = /D _STATIC_CPPLIB /D _DISABLE_DEPRECATE_STATIC_CPPLIB
 MS_RUNTIME_OPTION = $(MS_RUNTIME_OPTION) $(STATIC_CPPLIB_OPTION)
 !endif
-CXX_FLAGS=$(CXX_FLAGS) $(PLATFORM_WINMD) /AI"C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.15063.0" $(MS_RUNTIME_OPTION)
+CXX_FLAGS=$(CXX_FLAGS) $(PLATFORM_WINMD) /AI"C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.18362.0" $(MS_RUNTIME_OPTION)
 
 # How /GX option is spelled (Dirty Hack)
 GX_OPTION = /EHsc
diff --git a/hotspot/make/windows/makefiles/product.make b/hotspot/make/windows/makefiles/product.make
index 74d1a69e19c..e24dafde1f7 100644
--- a/hotspot/make/windows/makefiles/product.make
+++ b/hotspot/make/windows/makefiles/product.make
@@ -37,7 +37,7 @@ default:: $(BUILD_PCH_FILE) $(AOUT) checkAndBuildSA
 !include ../local.make
 !include compile.make
 
-PLATFORM_WINMD = /AI"E:\Programme\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.10.25017\lib\x86\store\references"
+PLATFORM_WINMD = /AI"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references"
 CXX_FLAGS=$(CXX_FLAGS) $(PLATFORM_WINMD) /EHsc $(PRODUCT_OPT_OPTION)
 
 RELEASE=
